ci/cron/src/Main.hs

```diff
@@ -106,11 +107,13 @@ docs_versions :: H.HashMap String String -> Set.Set String
 docs_versions json =
     Set.fromList $ H.keys json
 
-compare_versions :: GitHubVersion -> GitHubVersion -> Ordering
-compare_versions v1 v2 =
-    let t :: GitHubVersion -> [Integer]
-        t v = map read $ Split.splitOn "." $ name v
-    in compare (t v1) (t v2)
+newtype Version = Version (Int, Int, Int)
```

Comment:
Newtypes around tuples are usually not very useful. Either just make it a proper datatype, i..e, `data Version = Version Int Int Int` or use the tuple.